delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/StoragePluginConfig.scala

```diff
@@ -0,0 +1,24 @@
+package ch.epfl.bluebrain.nexus.delta.plugins.storage
+
+import ch.epfl.bluebrain.nexus.delta.plugins.storage.files.FilesConfig
+import ch.epfl.bluebrain.nexus.delta.plugins.storage.storages.StoragesConfig
+import com.typesafe.config.Config
+import pureconfig.generic.semiauto.deriveReader
+import pureconfig.{ConfigReader, ConfigSource}
+
+final case class StoragePluginConfig(storages: StoragesConfig, files: FilesConfig)
+
+object StoragePluginConfig {
+
+  /**
+    * Converts a [[Config]] into an [[StoragePluginConfig]]
+    */
+  def load(config: Config): StoragePluginConfig =
```

Comment:
Not a priority, could be done in 1.6 for example but we could handle config loading like for AppConfig